.buildkite/pipeline.yml

```diff
@@ -31,9 +31,19 @@ steps:
       manual:
         allowed: true
 
+  - label: "Serverless integration test"
```

Comment:
You should add a comment explaining why there is only a single test here. Otherwise nobody who reads this will know.